Common/Source/LKProfileResetDefault.cpp

```diff
@@ -266,6 +266,7 @@ void LKProfileResetDefault(void) {
   InverseInfoBox_Config=true; // black
   Appearance.InfoBoxModel=apImPnaGeneric;
 
+  Appearance.UTF8Pictorials = UTF8PICTORIALS;
```

Comment:
not same as Global_Init() ?

Reply:
Will change it, already prepared a rebased version after merge of NotamMultiselect.